frontend/packages/dev-console/integration-tests/features/Knative/knative-Actions_on_knative_revision.feature

```diff
@@ -3,72 +3,78 @@ Feature: Perform actions on knative revision
 
 Background:
    Given open shift cluster is installed with Serverless operator
-   And user is on dev perspective topology page
-   And one workload with knative resource is available
+   And user is at developer perspecitve
+   And open project namespace "aut-knative-actions-revision"
 
 
 @regression, @smoke
-Scenario: Knative revision menu options : Kn-03-TC01
-   Given knative revision name "nodejs-ex-git-1-q5rb8" is higlighted on topology page
-   When user right click on the knative revision
-   Then user able to see context menu with options "Edit Labels", "Edit Annotations", "Edit Revision", "Delete Revision"
+Scenario Outline: Knative revision menu options : Kn-03-TC01
+   Given knative service name "<service_name>" is higlighted on topology page
+   When user right click on the revision of knative service "<service_name>"
+   Then user able to see context menu with options Edit Labels, Edit Annotations, Edit Revision, Delete Revision
+
+Examples:
+    | service_name    | 
+    | nodejs-ex-git-2 |
 
 
 @regression
 Scenario: Edit labels popup details : Kn-03-TC02
-   Given knative revision name "nodejs-ex-git-1-q5rb8" is higlighted on topology page
+   Given knative service name "<service_name>" is higlighted on topology page
    When user selects "Edit Labels" option from knative revision context menu
    Then popup displays with header name "Edit Labels"
    And save button is disabled
 
 
 @regression, @smoke
 Scenario: Add label to the exisitng labels list : Kn-03-TC03
-   Given knative revision name "nodejs-ex-git-1-q5rb8" is higlighted on topology page
-   When user selects "Edit Labels" option from knative revision context menu
-   And add the label "app=label" to exisitng labels list in "Edit Labels" popup
-   And clicks "save" button on the "Edit Labels" popup
-   Then the label "app=label" display in side pane details
+   Given knative service name "nodejs-ex-git-2" is higlighted on topology page
+   When user right click on the revision of knative service "nodejs-ex-git-2" 
+   And user selects "Edit Labels" option from knative revision context menu
+   And add the label "app=label" to exisitng labels list in Edit Labels popup
+   And clicks save button on the "Edit Labels" popup
+   Then the label "app=label" display in "nodejs-ex-git-2" revision side pane details
 
 
 @regression
 Scenario: Remove label from exisitng labels list : Kn-03-TC04
-   Given knative revision name "nodejs-ex-git-1-q5rb8" is higlighted on topology page
+   Given knative service name "<service_name>" is higlighted on topology page
```

Comment:
as suggested above

Reply:
Revision name is auto generated, so we cannot assume the data.. Due to that reason I changed the step detail as knative service